client/components/section-header/test/index.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+/**
+ * Internal dependencies
+ */
+import SectionHeader from '../';
+
+describe( 'SectionHeader', () => {
```

Comment:
This is probably a good place [to use snapshot tests](https://jestjs.io/docs/en/snapshot-testing).

Reply:
Added a snapshot!!